### PR TITLE
[DNM] tests: fix/clean up upgrade/hammer-jewel-x

### PR DESCRIPTION
**Pull request by smithfarm**
- [ ] references tracker issue
- [ ] luminous backport

### CODE REVIEW

**Comment by smithfarm** [comment]
@yuriw Let's wait for @tchaikov but I certainly wouldn't complain if you did two runs of `upgrade/hammer-jewel-x (master)` - one without this PR, and another with it.

**Comment by smithfarm** [comment]
@yuriw Pushed `wip-20342` to Shaman.

**Comment by smithfarm** [comment]
Strike that - you can just do `--ceph master --suite upgrade/hammer-jewel-x --suite-repo https://github.com/smithfarm/ceph --suite-branch wip-20342`

File: qa/suites/upgrade/hammer-jewel-x/stress-split/1-hammer-install-and-upgrade-to-jewel/hammer-to-jewel.yaml [hunk 1]
```diff
@@ -63,6 +61,7 @@ upgrade-sequence-h-j:
   - exec:
       mon.a:
       - sleep 300
+      - ceph osd set sortbitwise
```
Comment (tchaikov) [inline qa/suites/upgrade/hammer-jewel-x/stress-split/1-hammer-install-and-upgrade-to-jewel/hammer-to-jewel.yaml]: can we reuse `qa/releases/jewel.yaml` instead, i.e.  add an extra step, like `1.5-jewel.yaml`, and link it to this file?

**Comment by smithfarm** [comment]
@tchaikov Makes sense, and I see more reconstructive surgery is needed here. Marking DNM for now.

**Comment by liewegas** [comment]
@smithfarm ping

**Comment by smithfarm** [comment]
Both hammer and jewel are EOL, so I'll just close this.

**Pull request closed without merging**